lib/worker.js

```diff
@@ -8,7 +8,7 @@ let worker;
 module.exports.startWorker = startWorker;
 module.exports.stopWorker = stopWorker;
 
-function startWorker() {
+function startWorker(createWorker) {
```

Comment:
```suggestion
function startWorker(createWorker = createWorker()) {
```

Reply:
I'm not quite sure how this would work? I didn't think default parameters could reference themselves

Reply:
Yeah, it makes no sense. I intended to make it:
```suggestion
function startWorker(createWorker = new Task(require.resolve('./xo-helper.js')) {
```

But thinking more about it, even that is not as readable.